File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zi=0
        for i in range(len(nums)):
            if (nums[i]!=0):
                nums[zi]=nums[i]
                zi+=1
        while (zi<len(nums)):
            nums[zi]=0
            zi+=1

File: 0283-move-zeroes/0283-move-zeroes.c
void moveZeroes(int* nums, int numsSize) {
    int zi=0;
    for(int i=0;i<numsSize;i++)
    {
        if (nums[i]!=0)
        {
            nums[zi]=nums[i];
            zi++;
        }
    }
    while (zi<numsSize)
    {
        nums[zi]=0;
        zi++;
    }
}